src/components/emby-scroller/emby-scroller.js

```diff
@@ -108,8 +108,7 @@ define(['scroller', 'dom', 'layoutManager', 'inputManager', 'focusManager', 'bro
             var headroom = new Headroom([], {
                 scroller: elem
             });
-            // initialise
-            headroom.init();
```

Comment:
why it is gone?

Reply:
I placed that code inside the function Headroom. So each time new Headroom is made, it "calls" init function that i removed here. The function attaches an event is css animation is supported in the browser. This way we dont need to call init. If this break something, we then see what's happening.